staff/joel-molero/cv-styled/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Curriculum Vitae</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header> 
+        <div class="child">
+        <h1 class="nombre">Joel Molero</h1>
+        </div>
+        <br>
+        <address>
+            Address: Spain, Barcelona, Febot street, Nº24, 2-1<br>
+            Tel: 93 710 63 28 || Mobile: 601 72 34 21<br>
+            Email: joel.molero@gmail.com<br>
+        </address>
+    <br>
+    </header>
+    <main>
+            <h1 class="title"><u>My Working Experience</u></h1>
+        <section class="working">
+            <br>
+            
+            <div class="workingExperience1">
+                <h3>2003-2007</h3>
+                <p class="experience">Lorem ipsum dolor sit amet consectetur adipisicing elit. Aliquam nam, at repellendus, enim quis voluptates facilis, in vitae earum natus ex est ipsam sunt ullam delectus eius maxime tempore nihil!</p>
```

Comment:
this class is not used in the css file, maybe it's not necessary to define it

Reply:
Fixed